lib/service/map_direction.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert';
+
+import 'package:concordi_around/model/direction.dart';
+import 'package:concordi_around/model/list_item.dart';
+import 'package:dio/dio.dart';
+import 'package:uuid/uuid.dart';
+
+import '../credential.dart';
+
+class MapDirection {
+
+  var uuid = Uuid();
+  String _sessionToken;
+
+    // Google Directions API requests
+  Future<List<ListItem>> _getDirection(String origin, String destination) async {
```

Comment:
You are missing a travel mode parameter

Reply:
You should add mode as a ordered optional parameter with a default value of 'car' as such: `_getDirection(String origin, String destination, [String mode = 'car'])`

Also double check this functions return type as you are not returning anything. Should it be void?

Reply:
Also double check this functions return type as you are not returning anything. Should it be void?